src/components/TodoItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  return (
+    <div data-cy="Todo" className={cn('todo', { completed: todo.completed })}>
+      {/* eslint-disable jsx-a11y/label-has-associated-control */}
+      <label htmlFor="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
```

Comment:
Consider using destruction assignment for todo object

Reply:
Not fixed